Update payment info section (#21702)
fleetdm/fleet: handbook/sales/README.md

For customers with large deployments, Fleet accepts payment via wire transfer or electronic debit (ACH/SWIFT).

> Payment information for customers within the United States is on Fleet's invoices so typically payment information will not need to be sent separately.

Reviewer: ```suggestion
Payment information for customers within the United States is on Fleet's invoices. Typically, payment information does not need to be sent separately.
```
